### DISCUSSION TITLE
Drop Shadow behind FloatingArrow component

### DISCUSSION THREAD

**Discussion by MoroseCorpse** [floating-ui/floating-ui#2992] [Q&A]
I would like to have a drop shadow for my tooltip. It works fine on my tooltip itself, but it doesn't work on the arrow. Here is an example: 
https://codesandbox.io/p/sandbox/example-arrow-vlzfh2

Unfortunately, you can also see the shadow above the tooltip. It's probably quite easy to fix, but I'm not that experienced in CSS and React and stuff like that yet

**Comment by baylorpaul** [comment]
I encountered a similar issue. While it may not be the most ideal solution, you could render the arrow twice, where the first arrow has a `zIndex: -1`.
E.g.
```
<FloatingArrow
	context={context}
	strokeWidth={1}
	style={{zIndex: -1, filter: 'drop-shadow(0 2px 4px rgba(0 0 0 / 0.4))'}}
/>
<FloatingArrow
	ref={arrowRef}
	context={context}
	fill="white"
	stroke="gray"
	strokeWidth={1}
/>
```
